src/hooks/screen/useChatScreenHook.ts

```diff
@@ -293,20 +297,12 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       }
       if (Platform.OS === 'android') {
         await currentChatSchema?.updateChatSentStatus(localDb, response);
-        initChatData();
+        handleChatData();
         refresh('channelList');
       } else {
-        queue.addPriorityJob({
-          operationLabel: DatabaseOperationLabel.ChatScreen_UpdateChatSentStatus,
-          id: `${selectedChannel?.id}-${randomId}`,
-          priority: QueueJobPriority.HIGH,
-          forceAddToQueue: true,
-          task: async () => {
-            await currentChatSchema?.updateChatSentStatus(localDb, response);
-            initChatData();
-            refresh('channelList');
-          }
-        });
+        await currentChatSchema?.updateChatSentStatus(localDb, response);
+        handleChatData();
+        refresh('channelList');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of `useDatabaseQueueHook` has been replaced with direct database operations and the newly introduced `handleChatData` function. This simplifies the code and makes it easier to understand. However, it's important to ensure that the removal of `useDatabaseQueueHook` does not affect the application's performance or functionality.
```diff
-        initChatData();
+        handleChatData();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->